deploy/docker/mcp_bridge.py

```
@@ -1,252 +1,442 @@
+        json_body = {}
+
+        # Check for unwrapped Pydantic body
+        if "_pydantic_body" in kwargs:
+            kwargs.pop("_param_name")  # Not needed in proxy
+            pydantic_data = kwargs.pop("_pydantic_body")
+            # FastAPI expects Pydantic model data as the root JSON body, not wrapped
+            json_body = pydantic_data
+        else:
+            # Process each parameter normally
+            for k, v in list(kwargs.items()):
+                placeholder = "{" + k + "}"
```

Ruff (0.13.1)

Local variable `placeholder` is assigned to but never used

Remove assignment to unused variable `placeholder`

(F841)